aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -1399,90 +1415,90 @@ TORCH_IMPL_FUNC(_convert_indices_from_csr_to_coo_structured_cpu)
  * https://github.com/scipy/scipy/blob/8a64c938ddf1ae4c02a08d2c5e38daeb8d061d38/scipy/sparse/sparsetools/csr.h
  * Modified to ensure sorted BSR column indices.
  */
-template <class I, class T>
-void _csr_to_block_csr_cpu_kernel(
-    const I n_row,
-    const I n_col,
-    const I R,
+template <class I, class T, bool compressed_rows>
```

Comment:
nit: ATen usually prefers `index_t` and `scalar_t`.

Reply:
Changed.